eZ/Publish/Core/REST/Client/Values/User/User.php

```diff
@@ -35,6 +35,14 @@ public function getVersionInfo()
     }
 
     /**
+     * {@inheritdoc}
```

Comment:
> Because inheritance is implicit it may happen that it is not necessary to include a PHPDoc with a "Structural Element". **This can cause confusion as it is now ambiguous whether the PHPDoc was omitted on purpose or whether the author of the code had forgotten to add documentation.**_
>
> In order to resolve this ambiguity the @inheritDoc tag can be used to indicate that this element will inherit its information from a super-element._
>
> Example:
>
> ```php
> /**
>  * This is a summary.
>  */
> class SuperClass 
> {
> }
> 
> /**
>  * @inheritDoc
>  */
> class SubClass extends SuperClass
> {
> }
> ```
>
> In the example above the SubClass' Summary can be considered equal to that of the SuperClass element, which is thus "This is a summary.".

Source: https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#61-making-inheritance-explicit-using-the-inheritdoc-tag written four years ago by @mvriel from PHPDocumentor team.

> Currently some applications have DocBlocks containing just the `{@inheritDoc}` inline tag to indicate that their complete contents should be inherited. This usage breaks with the PHPDoc Standard as summaries cannot contain inline tags and inheritance is automatic; you do not need to define a special tag for it._
>
> However, it does make clear that an element has been explicitly documented (and thus not forgotten). As such we are working to include a new (normal) tag in the PHPDoc Standard `@inheritDoc` that will serve that purpose._

Source: https://www.phpdoc.org/docs/latest/guides/inheritance.html#the-inheritdoc-tag

So to me its clear which direction this is going, even if that direction is taking "a wee bit of time" to propagate. And PHPStorm [afaik supports](https://youtrack.jetbrains.com/issue/WI-16547) this:

> Please note: @inheritDoc will be generated instead of {@inheritdoc} for the reasons explained here ...